scripts/globals/mobskills/astral_flow.lua

```diff
@@ -7,16 +7,34 @@ require("scripts/globals/status");
 require("scripts/globals/msg");
 ---------------------------------------------
 
+avatarOffsets={
```

Comment:
*friendly reminder that were brace-on-newline people here*

been letting it slide a lot but don't want it becoming habit for multi line things

Reply:
also, if you aren't using avatarOffsets outside this file, make it local